d3a_interface/matching_algorithms/__init__.py

```diff
@@ -0,0 +1,9 @@
+from .abstract_matching_algorithm import AbstractMatchingAlgorithm
+from .pay_as_bid_algorithm import PayAsBidMatchingAlgorithm
+from .pay_as_clear_algorithm import PayAsClearMatchingAlgorithm
+
+__all__ = [
+    AbstractMatchingAlgorithm,
+    PayAsBidMatchingAlgorithm,
+    PayAsClearMatchingAlgorithm
+]
```

Comment:
I think this should be a list of strings. From the [docs](https://docs.python.org/3.8/reference/simple_stmts.html):
> The public names defined by a module are determined by checking the module’s namespace for a variable named __all__; if defined, it must be a sequence of strings which are names defined or imported by that module.